install.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+# This script automates the installation of the cls3 tool.
+# It checks for the specified version (or fetches the latest one),
+# downloads the binary, and installs it on the system.
+
+# Check for required tools: curl and tar.
+# These tools are necessary for downloading and extracting the cls3 binary.
+if ! command -v curl &> /dev/null; then
+    echo "curl could not be found"
+    exit 1
+fi
+
+if ! command -v tar &> /dev/null; then
+    echo "tar could not be found"
+    exit 1
+fi
+
+# Determine the version of cls3 to install.
+# If no version is specified as a command line argument, fetch the latest version.
+if [ -z "$1" ]; then
+    VERSION=$(curl -s https://api.github.com/repos/go-to-k/cls3/releases/latest | grep -Po '"tag_name": "\K(.*?)(?=")')
```

Comment:
Since `grep -P` is not available on some environments, could you please do the following (or other ways)?

```suggestion
    VERSION=$(curl -s https://api.github.com/repos/go-to-k/cls3/releases/latest | grep '"tag_name":' | sed -E 's/.*"v([^"]+)".*/\1/')
```